openml/flows/functions.py

```diff
@@ -69,18 +72,20 @@ def _get_cached_flow(fid: int) -> OpenMLFlow:
                                    "cached" % fid)
 
 
-def get_flow(flow_id: int, reinstantiate: bool = False) -> OpenMLFlow:
+def get_flow(
+    flow_id: int,
+    extension: 'Extension' = None,
```

Comment:
the type hint shouldn't need to be a string literal because of the conditional import above, right?

Reply:
If it is not a string literal it will not work because then `Extension` will not be defined.